.github/workflows/build-deploy-book.yml

```diff
@@ -0,0 +1,59 @@
+name: build-deploy-book
```

Comment:
I'm a bit confused about the addition of this extra workflow - can you add a comment at the top describing what it is testing? Also rename to something more specific since `build-deploy-book` could apply to many of our other CI/CD jobs

Reply:
deleted -- that was vestigal -- I moved it into https://github.com/phaustin/jb_windows_test
and forgot to delete